agent/build.gradle.kts

```diff
@@ -76,10 +76,10 @@ tasks {
       )
     )
     projects = arrayOf(
-      rootProject, rootProject.project("agent"), rootProject.project("bootstrap"),
```

Comment:
[for reviewer] change was needed because there is no configuration for the root project and it now fails with the following message `Could not get unknown property 'licenseReport' for root project 'elastic-otel-java' of type org.gradle.api.Project`